R/loo_subsample.R

```diff
@@ -464,9 +464,47 @@ estimator_choices <- function() {
   c("hh_pps", "diff_srs", "srs")
 }
 
-
 ## Approximate elpd -----
 
+#' Utility function to apply user-specified log-likelihood to a single data point
```

Comment:
Oh, the problem is that `lpd_i` was a separate function instead of being an inner function in this commit. I think that this way is better, will bring it back

Reply:
Done, got the code back to its original state, `lpd_i` is now a separate function which I think is a cleaner approach than an inner function.

Reply:
Thats great!